packages/blockchain-wallet-v4-frontend/src/layouts/Wallet/MenuTop/Balance/BchWatchOnlyBalance/template.success.js

```diff
@@ -22,7 +22,7 @@ const Wrapper = styled.div`
 `
 
 const Success = props => {
-  return props.balance === 0 ? <div /> : (
+  return props.balance === 0 ? null : (
```

Comment:
I believe you could even have props.balance === 0 && (<LinkContainer...>...</LinkContainer>)